council/council.html

```diff
@@ -183,11 +182,11 @@ <h2 id="referral"><a href="#referral">Referral to W3C Council</a></h2>
     </ul>
   </li>
   <li>
-    <p>In extraordinary circumstances, the Council may vote to <a class="TODO" href="https://www.w3.org/Consortium/Process/Drafts/#council-delegation">delegate</a> or
+    <p>In exceptional circumstances, the Council may vote to <a class="TODO" href="https://www.w3.org/Consortium/Process/Drafts/#council-delegation">delegate</a> or
```

Comment:
Process says
> In extraordinary cases

So better to keep `extraordinary` here

Reply:
I suggested "exception" to match the note in https://www.w3.org/2023/Process-20230612/#council-short-circuit which is what this line is about.

Reply:
This is about [Extraordinary Delegation](https://www.w3.org/2023/Process-20230612/#council-delegation), so it makes sense to keep the same adjective, but both work.

Reply:
Ah you're right. Whichever you prefer, then!